pyaedt/edb_core/hfss.py

```diff
@@ -1024,6 +1024,84 @@ def create_lumped_port_on_net(
                 return edges_pts
         return True
 
+    @pyaedt_function_handler()
+    def create_vertical_circuit_port_on_clipped_traces(self, nets=None, reference_net=None, user_defined_extent=None):
+        """Create an edge port on clipped signal traces.
+
+        Parameters
+        ----------
+        nets : list, optional
+            List of nets, str or Edb net.
```

Comment:
I think we should clarify this. Is the data type a list or a string? If so, it should say so in the first line, with the parameter name. The description would then be something like this: "String of one net or EDB net or a list of multiple nets or EDB nets.

Reply:
Yes I agree. Done